### ISSUE TITLE
Typing mistakes or  incomplete code

### ISSUE THREAD

**Issue by ernestguevarra** [OxfordIHTM/codeditr#9]
ICD10:
* Minimum 4 characters is considered gold standard
* Number of 3 character codes are reported (but not considered incorrect)
* Number of 4 character codes are reported
* 2 character codes are counted and are considered incorrect

```
A & B	        Infectious and Parasitic Diseases
C	        Neoplasms
D	        Neoplasms, Blood, Blood-forming Organs
E	        Endocrine, Nutritional, Metabolic
F	        Mental and Behavioral Disorders
G	        Nervous System
H	        Eye and Adnexa, Ear and Mastoid Process
I	        Circulatory System
J	        Respiratory System
K	        Digestive System
L	        Skin and Subcutaneous Tissue
M	        Musculoskeletal and Connective Tissue
N	        Genitourinary System
O	        Pregnancy, Childbirth and the Puerperium
P	        Certain Conditions Originating in the Perinatal Period
Q	        Congenital Malformations, Deformations and Chromosomal Abnormalities
R	        Symptoms, Signs and Abnormal Clinical and Lab Findings
S	        Injury, Poisoning, Certain Other Consequences of External Causes
T	        Injury, Poisoning, Certain Other Consequences of External Causes
U	        no codes listed, will be used for emergency code additions
V, W, X, Y	External Causes of Morbidity (homecare will only have to code how patient was hurt; other settings will also code where injury occurred, what activity patient was doing)
Z	        Factors Influencing Health Status and Contact with Health Services (similar to current "V-codes")
```

**Comment by ernestguevarra** [comment]
```
#	Range	Chapter	#	Range	Chapter
1	1A00–1H0Z	Certain infectious or parasitic diseases	15	FA00–FC0Z	Diseases of the musculoskeletal system or connective tissue
2	2A00–2F9Z	Neoplasms	16	GA00–GC8Z	Diseases of the genitourinary system
3	3A00–3C0Z	Diseases of the blood or blood-forming organs	17	HA00–HA8Z	Conditions related to sexual health
4	4A00–4B4Z	Diseases of the immune system	18	JA00–JB6Z	Pregnancy, childbirth or the puerperium
5	5A00–5D46	Endocrine, nutritional or metabolic diseases	19	KA00–KD5Z	Certain conditions originating in the perinatal period
6	6A00–6E8Z	Mental, behavioural or neurodevelopmental disorders	20	LA00–LD9Z	Developmental anomalies
7	7A00–7B2Z	Sleep-wake disorders	21	MA00–MH2Y	Symptoms, signs or clinical findings, not elsewhere classified
8	8A00–8E7Z	Diseases of the nervous system	22	NA00–NF2Z	Injury, poisoning or certain other consequences of external causes
9	9A00–9E1Z	Diseases of the visual system	23	PA00–PL2Z	External causes of morbidity or mortality
10	AA00–AC0Z	Diseases of the ear or mastoid process	24	QA00–QF4Z	Factors influencing health status or contact with health services
11	BA00–BE2Z	Diseases of the circulatory system	25	RA00–RA26	Codes for special purposes
12	CA00–CB7Z	Diseases of the respiratory system	26	SA00–SJ3Z	Supplementary Chapter Traditional Medicine Conditions - Module I
13	DA00–DE2Z	Diseases of the digestive system	27	VA00–VC50	Supplementary section for functioning assessment
14	EA00–EM0Z	Diseases of the skin	28	XA0060–XY9U	Extension Codes
Unlike the [ICD-10](https://en.wikipedia.org/wiki/ICD-10) codes, the ICD-11 MMS codes never contain the letters I or O, to prevent confusion with the numbers 1 and 0.[[29]](https://en.wikipedia.org/wiki/ICD-11#cite_note-Refguide_1.2.7.1-29)
```

**Comment by ernestguevarra** [comment]
fixed by #40
